ansible/roles/ocp4-workload-rhte-analytics_data_ocp_workshop/tasks/workload.yml

```diff
@@ -198,23 +193,101 @@
     cluster_apps_domain: "{{ cluster_apps_domain_out.resources[0].spec.domain}}"
     notebook_suffix: "/notebooks/data-engineering-and-machine-learning-workshop.git/source/notebooks/hybrid-data-engineering.ipynb"
 
-- name: Insert user data into CSV file
-  lineinfile: 
-    path: /tmp/ausgeben.csv
-    line: "user{{ item }},openshift,{\"Notebook URL\":\"http://jupyterhub-opendatahub-user{{ item }}.{{ cluster_apps_domain }}/user/user{{ item }}{{ notebook_suffix }}\"}"
-  loop: "{{ range(1, num_users | int + 1, 1) | list }}"
+- name: make sure the guides project exists
+  k8s:
+    state: present
+    definition:
+      apiVersion: project.openshift.io/v1
+      kind: Project
+      metadata:
+        name: guides
+
+- name: search for username distribution tool
+  k8s_facts:
+    kind: DeploymentConfig
+    name: get-a-username
+    namespace: guides
+  register: r_gau_dc
+
+- name: deploy redis
+  when: r_gau_dc.resources | list | length == 0
+  shell: >
+    oc process -f -
+    -p DATABASE_SERVICE_NAME=redis
+    -p REDIS_PASSWORD=redis
+    -p VOLUME_CAPACITY=1Gi
+    -p MEMORY_LIMIT=1Gi
+    -p REDIS_VERSION=latest
+    | oc create -n guides  -f -
+  args:
+    stdin: "{{ lookup('file', './files/redis-template.yaml') }}"
```

Comment:
The role `./files/` directory is already in the standard file lookup path. This can be written more cleanly as:
```
    stdin: "{{ lookup('file', 'redis-template.yaml') }}"
```